docs/docs.html

```diff
@@ -530,6 +531,20 @@ <h5 id="config-details">Config Options</h5>
 									The encoding to use when opening local files. If specified, it must be a value supported by the FileReader API.
 								</td>
 							</tr>
+							<tr>
+								<td>
+									<code>decoder</code>
+								</td>
+								<td>
+									A function which allows custom decoding of the chunk.
```

Comment:
Could we explain what's the usage of this function. For me it is not clear when I should set the decoder option.

Reply:
Nodejs's Buffer support only a few encodings ('ascii','utf8','utf16le','ucs2','base64','latin1','binary','hex') while in the browser the fileReader supporting much more (https://encoding.spec.whatwg.org/) . 

when I tried to parse a stream with Papaparse I got "[ERR_UNKNOWN_ENCODING] Unknown encoding: SJIS" error because Buffer isn't supporting 'sjis' encoding.